include/library/spdm_crypt_lib.h

```diff
@@ -457,7 +458,7 @@ bool libspdm_asym_verify_ex(
     void* context,
     const uint8_t* message, size_t message_size,
     const uint8_t* signature, size_t sig_size,
-    uint32_t flags);
+    uint32_t *endian);
```

Comment:
The spdm_context is not passed into the libspdm_asm_verify functions that do endian swap and verification, so need an in\out parameter for endianness.